rust/headless-client/src/ipc_service/windows.rs

```diff
@@ -16,6 +26,65 @@ use windows_service::{
 const SERVICE_NAME: &str = "firezone_client_ipc";
 const SERVICE_TYPE: ServiceType = ServiceType::OWN_PROCESS;
 
+/// Returns true if the IPC service can run properly
```

Comment:
All this makes the file really long, but adding a refactoring would have made the diff huge.

There's too much IPC service code in `firezone-headless-client` so I think after this merges we should split that up.